pages/hydra/developer_resources/admin-users.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Make an Admin User"
```

Comment:
This page has the correct content, and I should've deleted it from admin-sets.md, which isn't currently linked from the navigation because it doesn't have real content yet.